src/parser/statement.js

```diff
@@ -913,6 +937,20 @@ pp.checkExport = function (node) {
   }
 };
 
+pp.checkDuplicateExports = function(node, name, isDefault) {
+  if (this.state.exportedIdentifiers[name]) {
```

Comment:
I think we need to manage `Object.prototype` here.

Something along the lines of this maybe?

```js
if (Object.getOwnPropertyNames(this.state.exportedIdentifiers).indexOf(name) !== -1) {
```